src/index.ts

```diff
@@ -365,34 +433,38 @@ export default class MondayConnector extends BaseHttpConnector<
     `)
   }
 
-  createWebhook(
-    boardId: number,
+  async createWebhook(
+    boardId: number | string,
     url: string,
     event: string,
-    config?: Record<string, unknown>,
-  ): Promise<MondayApiReponse['data']> {
-    return this.query(CREATE_WEBHOOK_QUERY, {
-      board_id: boardId,
-      url,
-      event,
-      config,
-    })
+    columnId?: string,
+  ): Promise<string> {
+    const params: any = { board_id: Number(boardId), url, event }
+    if (event === 'ChangeSpecificColumnValue') {
+      params.config = { columnId }
+    }
+    const res = await this.query(CREATE_WEBHOOK_QUERY, params)
+    return (res as any).create_webhook.id
```

Comment:
```suggestion
    return res?.create_webhook.id
```

Reply:
Changed to res! so that we don't fail silently

Reply:
Your lint doesn't like "!"s. tested explicitly